modules/govuk_jenkins/manifests/user.pp

```diff
@@ -0,0 +1,41 @@
+# == Class: govuk_jenkins::user
+#
+# Configure the Jenkins user and home directory
+#
+# === Parameters:
+#
+# [*home_directory*]
+#   The home directory of the user
+#
+# [*username*]
+#   The username you wish to use
+#
+class govuk_jenkins::user (
```

Comment:
This change specifically moves the management of this user from the base class to it's own class, and then includes it.